src/Pelagos/Bundle/AppBundle/Resources/public/css/remotelyhosted-datasets.css

```diff
@@ -0,0 +1,25 @@
+.cleair {
```

Comment:
Probably should be moved to entity.css, because this is used in other css files as well

Reply:
This cleair class is a pain, it's the OLD OLD style of the site.  I vote for no style, and yes, refactor.

Reply:
No style looks very pale and we already have the colorful noty message. Moving to entity.css requires to do more testing in this case (overlapping css styles, same class name different attributes/values). I can do a refactor after this.